tests/template_backends/test_dummy.py

```diff
@@ -55,6 +56,25 @@ def test_html_escaping(self):
         self.assertIn('&lt;script&gt;', content)
         self.assertNotIn('<script>', content)
 
+    def test_django_html_escaping(self):
+        if self.backend_name == 'dummy':
+            return
```

Comment:
I just did it this way because `test_get_template_syntax_error` does it, too.
I'll change both.